types/mapcmd/map.go

```diff
@@ -48,6 +75,35 @@ func getMap(ctx context.Context, clientConfig *hazelcast.Config, mapName string)
 	return
 }
 
+func NewUse() *cobra.Command {
+	//var mapName, mapKey string
+	cmd := &cobra.Command{
+		Use:   "use map-name",
+		Short: "sets default map name",
+		Example: "map use m1		# sets the default map name to m1 unless set explicitly\n" +
```

Comment:
You defined constants for map put/get examples. Maybe you would like to do that also for this one for consistency.

Reply:
Is there a reason you use the tab character in some strings and spaces in others?

Reply:
All tabs are converted to spaces and map use example is referenced in map command on https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/a23e04ffb4abf7bc4d4b80b6b7906acae7409a75